examples/manifest.toml

```diff
@@ -0,0 +1,94 @@
+#:schema ./schema.json
```

Comment:
New manifest format is a lot simpler, just a single `categories` key, each of which contains a `title`, `prelude`, and a list of `examples`.

The parser will automatically infer which languages are available based on the filesystem structure. It will include any examples with frontmatter at `examples/{LANGUAGE}/{NAME}/README.md` (where `LANGUAGE` is one of `cpp`, `python`, `rust`).